pkg/api/v1/clients/client_interface.go

```diff
@@ -178,3 +178,20 @@ func (o WatchOpts) WithDefaults() WatchOpts {
 	}
 	return o
 }
+
+func TranslateWatchOptsIntoListOpts(wopts WatchOpts) ListOpts {
```

Comment:
nit: can we move these from this file to one that has a different name. It feels like this is the right package but client_interface as a file is not where I would expect to find it .

Perhaps it could be combined with the current function in options and turned into a utilities file?

Reply:
I am good with that